ProjFS.Mac/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -210,6 +210,11 @@ static VirtualizationRootHandle FindOrDetectRootAtVnode(vnode_t _Nonnull vnode,
     {
         PrjFSVirtualizationRootXAttrData rootXattr = {};
         SizeOrError xattrResult = Vnode_ReadXattr(vnode, PrjFSVirtualizationRootXAttrName, &rootXattr, sizeof(rootXattr));
+        if (xattrResult.error == ENOATTR)
+        {
+            xattrResult = Vnode_ReadXattr(vnode, PrjFSVirtualizationRootXAttrNameOld, &rootXattr, sizeof(rootXattr));
+        }
+        
```

Comment:
Key question about this PR is whether to bother with fallbacks like this, or if we want any other kind of backwards compatibility facility, such as a script/tool that you can run over your VFS4G repos to rename all the xattrs.

Reply:
My 2c: We only have a few early adopters who are actively using VFS4G repos on MacOS. They're ready for some short term pain, so I don't think it's too unreasonable to ask them to push any local changes, delete their current clones and reclone with the new version whenever they pick this change up. So long as this is one of the only times we have to make a change like this, it seems fine to me.

If we do that, we can also avoid putting the old version of the xattr into the code. None of our automation relies on long lived repos, we clone a fresh one for every test.

I'm curious to hear what everyone else thinks.